dirs/dirs.go

```diff
@@ -262,6 +263,7 @@ func SetRootDir(rootdir string) {
 	AppArmorCacheDir = filepath.Join(rootdir, "/var/cache/apparmor")
 	SnapAppArmorAdditionalDir = filepath.Join(rootdir, snappyDir, "apparmor", "additional")
 	SnapDownloadCacheDir = filepath.Join(rootdir, snappyDir, "cache")
+	SnapSeccompBase = filepath.Join(rootdir, snappyDir, "seccomp")
 	SnapSeccompDir = filepath.Join(rootdir, snappyDir, "seccomp", "bpf")
```

Comment:
(nitpick) this could now be written as: `SnapSeccompDir = filepath.Join(SnapSeccompBase, "bpf")` - but only worth fixing if the PR is not green already :)